.gitignore

```diff
@@ -1,4 +1,3 @@
 target
-Cargo.lock
 .idea
 *.iml
```

Comment:
```suggestion
.DS_Store
```

Reply:
I don't think `.DS_Store` belongs in every single `.gitignore` and should be global.

Reply:
Sure. We're not likely to get any contributions to this repo, so I guess it's fine to assume they'll be competent.